src/common/k8s-api/kube-api.ts

```diff
@@ -295,14 +295,18 @@ export class KubeApi<T extends KubeObject> {
    */
   private async getLatestApiPrefixGroup() {
     // Note that this.options.apiBase is the "full" url, whereas this.apiBase is parsed
-    const apiBases = [this.options.apiBase, ...this.options.fallbackApiBases];
+    const apiBases = [this.options.apiBase, this.objectConstructor.apiBase, ...this.options.fallbackApiBases];
```

Comment:
All these `apiBase`'s are marked as optional so that is why we need to skip empty `apiUrl`'s below.

Reply:
would it be better to filter empty urls here?

Reply:
You mean instead of in the `for` loop below? I don't really see a difference but I can if you think it would read better.

Reply:
Yeah, it's not a big deal, both are fine. 

Code looks good, don't know how to reproduce and test it. I can see ingresses on my clusters.

Reply:
I would try it out with an older version of kube, one before 1.19